modules/reference/pages/properties/cluster-properties.adoc

```diff
@@ -3740,6 +3847,22 @@ Election timeout expressed in milliseconds.
 
 ---
 
+=== raft_enable_longest_log_detection
+
+Enables additional step in leader election where candidate is allowed to wait for all the replies from node it requested votes from. This may introduce a small delay when recovering from failure but will prevent truncation if any of the replicas has more data than the majority.
```

Comment:
```suggestion
Enables an additional step in leader election where a candidate is allowed to wait for all the replies from the node it requested votes from. This may introduce a small delay when recovering from failure but will prevent truncation if any of the replicas have more data than the majority.
```